### DISCUSSION TITLE
Netbox Legal T&C

### DISCUSSION THREAD

**Discussion by aquelf** [netbox-community/netbox#6485] [Q&A] [answered]
https://github.com/netbox-community/netbox/blob/develop/LICENSE.txt

Is this the official T&C for Netbox? If not could somebody point me in the right direction?

Many thanks

**Comment by jeremystretch** [comment, marked as answer]
Yes, that is the license under which NetBox is published. It is a standard open source license referred to as [Apache 2](https://en.m.wikipedia.org/wiki/Apache_License).

**Reply by aquelf** [reply]
Great thanks Jeremy.

Appreciate the prompt response,
